src/share/qtcreator/externaltools/lupdate.xml

```diff
@@ -31,10 +31,11 @@
     <displayname>Update Translations (lupdate)</displayname>
     <category>Linguist</category>
     <order>1</order>
-    <executable>
-        <path>%{CurrentProject:QT_INSTALL_BINS}/lupdate</path>
+    <executable modifiesdocument="yes">
```

Comment:
It was suspicious to me that in the original code modifiesdocument is not set, so I checked it and it turned out that the name modifiesdocument is not very self-explanatory - what it means is "modifies current document", so we should leave the default "no" here. https://github.com/sailfishos/sailfish-qtcreator/blob/master/src/plugins/coreplugin/externaltool.cpp#L392

Reply:
It is theoretically possible that it modifies the current document - if one of the language files happen to be opened, which I guess is pretty much never. If you are happy with the default, I can just remove it.